src/Microsoft.DotNet.Build.Tasks/project.json

```diff
@@ -1,15 +1,16 @@
 {
   "dependencies": {
-    "System.Reflection.Metadata": "1.3.0-rc3-24128-00",
-    "System.Dynamic.Runtime": "4.0.11-rc3-24128-00",
-    "System.Xml.XPath.XmlDocument": "4.0.0",
     "Microsoft.Build.Framework": "0.1.0-preview-00022",
     "Microsoft.Build.Utilities.Core": "0.1.0-preview-00022",
     "Microsoft.Build.Tasks.Core": "0.1.0-preview-00022",
+    "NETStandard.Library": "1.5.0-rc2-24027",
     "Newtonsoft.Json": "7.0.1",
+    "NuGet.Packaging": "3.5.0-beta2-1456",
     "NuGet.Versioning": "3.5.0-beta2-1456",
     "NuProj": "0.10.4-beta-gf7fc34e7d8",
-    "NETStandard.Library": "1.5.0-rc2-24027"
+    "System.Reflection.Metadata": "1.3.0-rc3-24128-00",
+    "System.Dynamic.Runtime": "4.0.11-rc3-24128-00",
```

Comment:
Haha,  I only touched this file to add nuget.packaging and to put these entries into alphabetical order but it appears I didn't get the alphabetical order bit entirely correct.  I'll submit a tweak for this tomorrow.